package.json

```diff
@@ -31,35 +31,6 @@
     "stylelint-scss": "^2.2.0",
     "stylelint-selector-bem-pattern": "^2.0.0"
   },
-  "browserslist": [
-    "last 2 versions",
-    "ie 8",
-    "ie 9",
-    "android 2.3",
-    "android 4",
-    "opera 12"
-  ],
-  "gulpPaths": {
```

Comment:
There should be defaults declared somewhere; at minimum to make it easier to build a custom manifest.  But ideally also being used as a fallback for any (new/future) settings that may not be defined in the project-specific manifest (yet).

Reply:
I wanted to define defaults in the module manifest (see second paragraph).

Reply:
Yes, that was already clear from the description.  However, there is currently no way to find out how such a manifest file would look like and how parameters need to be specified.  An `example.manifest.json` would be helpful.  A fallback manifest even better.

Reply:
@fabianmarz We should leave the browsers list in `package.json` because Autoprefixer looks there for it - https://github.com/ai/browserslist

Reply:
Hm, that somewhat questions the sense of this whole PR, doesn't it?

It's not really clear to me what actual problem we have with these settings in `package.json` other than being there / "polluting" the file?

Having some settings here and some others elsewhere would be inconsistent.

Reply:
Yes, you're right, I wasn't aware of this. Thanks for pointing out. Also there was a wrong path given. 🙈If I understand this correctly, you can omit the argument and the module will look for the browsers itself.

Reply:
@sun, indeed. We should not split the config into several files. I'll revert the changes.